pages/lobby_page.py

```diff
@@ -55,9 +56,14 @@ def create_room(self):
         self.context.wait.until(EC.visibility_of_element_located((By.ID, 'create-room-name')))
 
     def find_btn(self):
+        if "lobby" not in self.context.driver.current_url:
+            self.context.driver.get(self.context.base_url + "/chat/lobby")
+            self.context.wait.until(lambda driver: driver.find_element_by_id('status_dropdown'))
```

Comment:
Do it works with EC, not lambda ?

Reply:
Unfortunately not, this expectation we used from the very beginning, this would wait for the download of the chat and it is stably work.
If you have some stable work ulternative I with pleaser add it here.